lms/lib/comment_client/utils.py

```diff
@@ -171,3 +172,19 @@ def __init__(self, collection, page, num_pages, thread_count=0, corrected_text=N
         self.num_pages = num_pages
         self.thread_count = thread_count
         self.corrected_text = corrected_text
+
+
+def check_forum_heartbeat():
+    """
+    Check the forum connection via its built-in heartbeat service and create an answer which can be used in the LMS
+    heartbeat django application.
+    This function can be connected to the LMS heartbeat checker through the HEARTBEAT_CHECKS variable.
+    """
+    try:
+        res = requests.get('%s/heartbeat' % COMMENTS_SERVICE).json()
```

Comment:
The Forum can be set up to require an API key.  Is the heartbeat URL always accessible without authentication?

Reply:
Yes. There's a particular exception for the heartbeat URL: https://github.com/edx/cs_comments_service/blob/71f046b11e3ad79fc8ab412bf1e8a15dcb1e0183/app.rb#L86